ding_availability.module

```diff
@@ -183,9 +184,18 @@ function ding_availability_holdings($provider_ids) {
         'class' => array('availability-holdings-table'),
       );
 
+      // check if there are items in acquisition
+      foreach ($item['holdings'] as $holding) {
+        $item['ordered_count'] += $holding[ordered_count];
```

Comment:
`ordered_count` should be marked as a string e.g. `$holding[ordered_count]`.

Also I would expect a check to see if `$holding[ordered_count]` is defined (or not empty). Otherwise this will cause a notice. I do not expect that all providers support "ordered_count"?